src/locale.rs

```diff
@@ -20,26 +20,29 @@ use std::env;
 use objc::runtime::{Class, Object};
 
 pub fn set_locale_environment() {
-    let locale_id = unsafe {
+    let (language_code, country_code) = unsafe {
         let locale_class = Class::get("NSLocale").unwrap();
         let locale: *const Object = msg_send![locale_class, currentLocale];
         msg_send![locale_class, release];
-        let identifier: *const Object = msg_send![locale, localeIdentifier];
+        let language_code: *const Object = msg_send![locale, languageCode];
+        let country_code: *const Object = msg_send![locale, countryCode];
         msg_send![locale, release];
-        let identifier_str = nsstring_as_str(identifier).to_owned();
-        msg_send![identifier, release];
-        identifier_str
+        let language_code_str = nsstring_as_str(language_code).to_owned();
+        msg_send![language_code, release];
+        let country_code_str = nsstring_as_str(country_code).to_owned();
+        msg_send![country_code, release];
+        (language_code_str, country_code_str)
     };
-    let locale_id = locale_id + ".UTF-8";
+    let locale_id = format!("{}_{}.UTF-8", &language_code, &country_code);
     env::set_var("LANG", &locale_id);
-    env::set_var("LC_CTYPE", &locale_id);
+    // env::set_var("LC_CTYPE", &locale_id);
```

Comment:
This something that should be deleted?

Reply:
er yeah

Reply:
I tried both, and it didn't seem to make a difference. iTerm doesn't appear to set it, so I'm opting to leave it out, but I'd be happy to put it back in.

Reply:
I'm not a Mac user anymore, so I can't really weigh in here, but copying iTerm seems reasonable.